contracts/polygon/connectors/aave/v3-import/main.sol

```diff
@@ -92,6 +92,88 @@ contract AaveV3ImportResolver is AaveHelpers {
 		);
 	}
 
+	function _importAaveWithCollateral(address userAccount, ImportInputData memory inputData, bool[] memory enableCollateral)
+		internal
+		returns (string memory _eventName, bytes memory _eventParam)
+	{
+		require(
+			AccountInterface(address(this)).isAuth(userAccount),
+			"user-account-not-auth"
+		);
+
+		require(inputData.supplyTokens.length > 0, "0-length-not-allowed");
```

Comment:
@Richa-iitr Can you check if `enableCollateral` length eq to `supplyTokens` length in another require statement?

Reply:
okay

Reply:
done